micromamba/tests/test_create.py

```diff
@@ -652,6 +652,23 @@ def test_spec_with_channel_and_subdir():
         )
 
 
+def test_spec_with_slash_in_channel():
```

Comment:
This would make sure the test is properly isolated when running locally.
```suggestion
def test_spec_with_slash_in_channel(tmp_home, tmp_root_prefix):
```